node/testing/src/server/simulator.rs

```diff
@@ -85,9 +85,11 @@ async fn simulation_create(
             while let Some(cluster_mutex) = cluster_mutex.upgrade() {
                 let mut cluster = cluster_mutex.lock().await;
                 let mut runner = ClusterRunner::new(&mut cluster, |_| {});
-                let _ =
-                    tokio::time::timeout(Duration::from_millis(500), simulator.run(&mut runner))
-                        .await;
+                let _ = tokio::time::timeout(
+                    Duration::from_millis(500),
+                    simulator.setup_and_run(&mut runner),
```

Comment:
this change is wrong and will break the webnodes custom cluster setup.
It will cause simulator to be `setup` over and over, which is not the goal.

Reply:
tbh I am not even sure what the change is about

Reply:
Yeah idk. @vlad9486 ?

Reply:
Yes, it should be just `run` without `setup`.